components/habitat-chef-io/content/habitat/environment-variables.md

```diff
@@ -41,7 +41,8 @@ This is a list of all environment variables that can be used to modify the opera
 | `HAB_STUDIO_NOSTUDIORC` | build system | no default | When set to a non-empty value, a `.studiorc` will not be sourced when entering an interactive Studio via `hab studio enter`. |
 | `HAB_STUDIO_SUP` | build system | no default | Used to customize the arguments passed to an automatically launched Supervisor, or to disable the automatic launching by setting it to `false`, `no`, or `0`. |
 | `HAB_GLYPH_STYLE` | build system | `full` (`limited` on Windows) | Used to customize the rendering of unicode glyphs in UI messages. Valid values are `full`, `limited`, or `ascii`. |
-| `HAB_UPDATE_STRATEGY_FREQUENCY_MS` | Supervisor | 60000 | Frequency of milliseconds to check for updates when running with an [update strategy](/docs/using-habitat#using-updates) |
+| `HAB_SUP_UPDATE_MS` | Supervisor | 60000 | Frequency in milliseconds governing how often to check for Supervisor updates when running with an [update strategy](/docs/using-habitat#using-updates) |
+| `HAB_UPDATE_STRATEGY_FREQUENCY_MS` | Supervisor | 60000 | Frequency in milliseconds governing how often to check for Service updates when running with an [update strategy](/docs/using-habitat#using-updates) |
```

Comment:
```suggestion
| `HAB_UPDATE_STRATEGY_FREQUENCY_MS` | Supervisor | 60000 | Frequency in milliseconds governing how often to check for service updates when running with an [update strategy](/docs/using-habitat#using-updates) |
```

Reply:
Thanks @kagarmoe. I've handled in https://github.com/habitat-sh/habitat/commit/a4ca1bcde50925aaa02fbd8f2fecf8a0bd5624d4